pyproject.toml

```diff
@@ -9,7 +9,11 @@ requires = [
     # wheels on PyPI
     #
     # see: https://github.com/scipy/oldest-supported-numpy/blob/main/setup.cfg
-    "oldest-supported-numpy",
+    "oldest-supported-numpy; python_version!='3.10' or platform_system!='Windows' or platform_python_implementation=='PyPy'",
+    # For CPython 3.10 under Windows, SciPy requires NumPy 1.22.3 while the
+    # oldest supported NumPy is defined as 1.21.6. We therefore need to force
+    # it for this specific configuration.
```

Comment:
From reading the issue, this was copied from SciPy's `pyproject.toml`. For future maintainers, lets reference it:

```suggestion
    # it for this specific configuration. For details, see
    # https://github.com/scipy/scipy/blob/c58b608c83d30800aceee6a4dab5c3464cb1de7d/pyproject.toml#L38-L41
```